data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -130,35 +103,19 @@ public void getLineageInfo() throws ParseException, SemanticException {
 		Dispatcher disp = new DefaultRuleDispatcher(this, rules, null);
 		GraphWalker ogw = new DefaultGraphWalker(disp);
 		// Create a list of topop nodes
-		ArrayList<org.apache.hadoop.hive.ql.lib.Node> topNodes = new ArrayList<org.apache.hadoop.hive.ql.lib.Node>();
+		List<org.apache.hadoop.hive.ql.lib.Node> topNodes = new ArrayList<org.apache.hadoop.hive.ql.lib.Node>();
 		topNodes.add(tree);
 		ogw.startWalking(topNodes, null);
 	}
 
-//	private ASTNode search(int type, ASTNode node) {
-//		System.out.println("searching node = " + node);
-//		if (node != null) {
-//			if (node.getType() == type) {
-//				return node;
-//			} else {
-//				for (int i = 0; i < node.getChildCount(); i++) {
-//					ASTNode foundNode = search(type, (ASTNode) node.getChild(i));
-//					return foundNode;
-//				}
-//			}
-//		} else {
-//			return null;
-//		}
-//		return null;
-//	}
 
 	/**
 	 * Implements the process method for the NodeProcessor interface.
 	 */
+	@Override
 	public Object process(org.apache.hadoop.hive.ql.lib.Node nd, Stack<org.apache.hadoop.hive.ql.lib.Node> stack, NodeProcessorCtx procCtx,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "process" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)